sigs.yaml

```
@@ -82,6 +82,28 @@ sigs:
         - name: userinterface
           owners:
             - https://raw.githubusercontent.com/kubevirt/web-ui/master/OWNERS
+    - dir: kubevirt-sig-scale
```

Can we rename this accordingly so that it matches the style of the other sig names?
```suggestion
    - dir: sig-scale
```

---

> Can we rename this accordingly so that it matches the style of the other sig names?

Background is the assignment of test cases, where this would then align with the rest of the assignments.